.github/workflows/docker.yml

```diff
@@ -30,14 +34,14 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v2
 
-      - name: Build and push to <insert repo>
+      - name: Build and push to quay.io/stevebs/heatcluster
```

Comment:
I think you have a typo here

Your repository name is quay.io/stevebs22/heatcluster (https://quay.io/repository/stevebs22/heatcluster)

Reply:
Fixed this. Thanks!